sdks/go/pkg/beam/runners/dataflow/dataflow.go

```diff
@@ -181,11 +183,15 @@ func Execute(ctx context.Context, p *beam.Pipeline) (beam.PipelineResult, error)
 	if err != nil {
 		return nil, err
 	}
-	enviroment, err := graphx.CreateEnvironment(ctx, jobopts.GetEnvironmentUrn(ctx), getContainerImage)
+	environment, err := graphx.CreateEnvironment(ctx, jobopts.GetEnvironmentUrn(ctx), getContainerImage)
 	if err != nil {
 		return nil, errors.WithContext(err, "generating model pipeline")
 	}
-	model, err := graphx.Marshal(edges, &graphx.Options{Environment: enviroment})
+	model, err := graphx.Marshal(edges, &graphx.Options{Environment: environment})
+	if err != nil {
+		return nil, errors.WithContext(err, "generating model pipeline")
+	}
+	err = pipelinex.ApplySdkImageOverrides(model, jobopts.GetSdkImageOverrides())
 	if err != nil {
 		return nil, errors.WithContext(err, "generating model pipeline")
```

Comment:
```suggestion
		return nil, errors.WithContext(err, "applying container image overrides")
```

Reply:
Done, also updated the one a few lines above.